src/jit-diff/jit-diff.cs

```diff
@@ -291,11 +291,11 @@ public T ExtractDefault<T>(string name, out bool found)
 
             private void LoadFileConfig()
             {
-                _jitDasmRoot = Environment.GetEnvironmentVariable("JIT_DASM_ROOT");
+                _jitUtilsRoot = Environment.GetEnvironmentVariable("JIT_UTILS_ROOT");
 
-                if (_jitDasmRoot != null)
+                if (_jitUtilsRoot != null)
                 {
-                    string path = Path.Combine(_jitDasmRoot, "asmdiff.json");
+                    string path = Path.Combine(_jitUtilsRoot, "asmdiff.json");
```

Comment:
Should probably change the name asmdiff.json to something like jitutils.json to cover the broader use.